content/blog/2020-10-12-october-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,162 @@
+---
+title: October '20 Heartbeat
+date: 2020-10-12
+description: |
+  This month, hear about our international talks, new video docs, 
+  and the best tutorials from our community.
+descriptionLong: |
+  This month, hear about our international talks, new video docs, 
+  and the best tutorials from our community.
+picture: 2020-10-12/cover.png
+pictureComment:
+  Double DeeVee! One of these birds is on a layover before heading to Germany.
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/october-20-heartbeat/527
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - Google Drive
+  - Tutorials
```

Comment:
we had some tags for conferences? meetups, etc ... should we make one for YouTube btw?